ch_tools/chadmin/cli/data_store_group.py

```diff
@@ -374,8 +374,11 @@ def detect_broken_partitions(ctx, root_path, reattach, detach):
         objects: List[S3ObjectLocalInfo] = []
         logging.debug(f"Checking files from: {path}")
         for file in files:
-            file_full_path = Path(os.path.join(path, file))
-            objects.extend(S3ObjectLocalMetaData.from_file(file_full_path).objects)
+            try:
+                file_full_path = Path(os.path.join(path, file))
+                objects.extend(S3ObjectLocalMetaData.from_file(file_full_path).objects)
+            except Exception as e:
```

Comment:
Can we handle more specific exception? `Exception` is too general.

And why 
```
 file_full_path = Path(os.path.join(path, file))
```
operation must be in try?

Reply:
What problem will be resolved if we use a more specific exception?

Reply:
For example, a broken S3 connection will result in overloaded logs, a lot of junky s3 requests. 
And we also can't imagine all such cases, so better to handle only what we already know.
We can use `keep-going` if you want to ignore all exceptions.

Reply:
If it is a problem what quantities of extra logs/requests are we talking about?